Cargo.toml

```diff
@@ -9,11 +9,12 @@ license = "MIT/Apache-2.0"
 description = "Yet another unofficial implementation of Apache ORC spec in safe Rust"
 keywords = ["arrow", "orc", "arrow-rs"]
 include = ["src/**/*.rs", "Cargo.toml"]
+rust-version = "1.70"
```

Comment:
for the error
```
error: package `half v2.3.1` cannot be built because it requires rustc 1.70 or newer, while the currently active rustc version is 1.69.0
Either upgrade to rustc 1.70 or newer, or use
```